modules/netstat/netstat.go

```diff
@@ -84,6 +86,11 @@ type statistics struct {
 	Totalhits float64 `json:"totalhits"`
 }
 
+// Indicates module should use namespace resolution mode, which changes
+// the behavior on Linux; optional right now but may be the default in
+// the future.
+var namespaceMode bool
```

Comment:
That's correct, I just didn't want to initially replace the existing method. My thoughts were we support both and remove the previous method later.